src/http-server.js

```diff
@@ -7,6 +7,11 @@ import { telemetry } from './telemetry.js'
 import inspect from './inspect/index.js'
 import { version } from './util.js'
 
+const checkReadinessSampling = [
+  config.readinessDynamoCheckSampling,
+  config.readinessS3CheckSampling
+]
```

Comment:
I think these should be broken out as individual parameters instead of the array.  In the current implementation, the ordering of this array here _must_ match the order of the checks in the health-check.js implementation (ie dynamo check first, s3 check second).  It would be hard to catch an error if we happened to switch these (or the order of the readiness array), or if we've added another check but didn't change this sample array to match.